te3/tutorials/incremental-refresh-about.md

```diff
@@ -22,6 +22,10 @@ Datasets hosted in the Power BI service can have [Incremental Refresh](https://l
 
 _Incremental refresh can be easily configured and modified from within Tabular Editor._
 
+> [!IMPORTANT]
+> Setting up this type of Incremental Refresh with Tabular Editor 3 is limited to dataset hosted in the Power BI Datasets service.
```

Comment:
Slight change in the language for clarity -- 
I think it should be "Setting up Incremental Refresh with Tabular Editor 3 is limited to datasets hosted in the Power BI Service."

Reply:
That was also my original version, but I then though that there are other forms of incremental refresh where you set it all up yourself, e.g. in a Analysis Service datamodel, and I didn't want to imply that this is the only form of incremental refresh possible... Not sure what is better since we do not, to my knowledge, have a section about setting up "old fashioned" incremental refresh

Reply:
Fair point; maybe we could refer to AS incremental refresh in a separate block either nearby or the bottom of the page? What do you think?

Reply:
Yes, that is a good idea. Was thinking of having a link there to some other walkthrough, I just couldn't find anything good to link to. 

@otykier Do you know of a good resource (perhaps external) that talks about setting up incremental refresh on AS models?

Reply:
@data-goblin 
I have made a few tweaks given the discussion. Changed the wording, added a link to learn on partitions and moved the section slightly to make the content flow more easily.